### ISSUE TITLE
Any way to have this working on < API 31 on Android?

### ISSUE THREAD

**Issue by javiercampos** [Redth/ZXing.Net.Maui#54]
I can't seem to get the camera on Android to work on less than API 31 (Android 12), neither in real devices nor emulators... I'd specifically need Android 11, but wondering what is the limiting factor, and if I could make my own compilation (if it's a "release" problem) for API 30

**Comment by javiercampos** [comment]
Okay, never mind... it seems to work on API 30, however the camera viewing control does not show when it's within a `<Border>` control (it works on Android 12).

Also, because of the inclusion of the `Xamarin.AndroidX.Camera.*` packages, you get a ton of CA1416 warnings (`
Warning	CA1416	This call site is reachable on: 'Android' 30.0 and later. 'CameraViewExtensions.UseBarcodeReader(MauiAppBuilder)' is only supported on: 'Android' 31.0 and later.`) when compiling for any version under 31...

... however, it seems to work in runtime, so I'll just suppress the warnings for the time being

**Comment by dodoqang** [comment]
I also encountered this problem. Android 11 can't be used either
message：
10-05 04:03:42.655	pixel_4_-_api_30	Error	420	Camera3-OutputStream	getBufferLockedCommon: Stream 1: Can't dequeue next output buffer: Connection timed out (-110)

**Comment by chihabhajji** [comment]
my physical device is running on Android 8 and i cant seem to be able to run and debug on an emulator

**Comment by knocte** [comment]
> so I'll just suppress the warnings for the time being

Hey @javiercampos how did you supress the warning? Does it have a specific warning code?

**Comment by knocte** [comment]
Mmm yeah I know about pragmas, but my warning has happening in a XAML file 😬 
Thanks for the reply though.

**Comment by javiercampos** [comment]
> Mmm yeah I know about pragmas, but my warning has happening in a XAML file 😬 Thanks for the reply though.

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/errors-warnings#nowarn

**Comment by knocte** [comment]
That's for the `.?sproj` file I assume (XML, vs XAML) 😄  I could do that, but then I would ignore this warning for the whole app instead of in the specific place where I use the ZXing XAML tag 😭

**Comment by javiercampos** [comment]
> That's for the `.?sproj` file I assume (XML, vs XAML) 😄 I could do that, but then I would ignore this warning for the whole app instead of in the specific place where I use the ZXing XAML tag 😭

Yes, that'd remove the whole analyzer warning for all the app... I'm not sure there's a way to specifically remove it only for this
